mlkem/poly_k.c

```diff
@@ -205,6 +205,11 @@ void polyvec_tomont(polyvec *r)
  *
  * Arguments:   - poly *r: pointer to output polynomial
  *              - const uint8_t *buf: pointer to input byte array
+ *
+ * Specification: Implements [FIPS 203, Algorithm 8, SamplePolyCBD_eta1], where
```

Comment:
Algorithm8 is for both eta1 and eta2, right?

Reply:
yes, but here we implement a specific instance of it